packages/augur-core/source/contracts/legacy_reputation/OldLegacyRepToken.sol

```diff
@@ -107,8 +107,8 @@ contract OldLegacyReputationToken is DelegationTarget, ITyped, Initializable, Va
     }
 
     function transferFrom(address _from, address _to, uint _value) public whenNotMigratingFromLegacy returns (bool) {
-        require(_value <= _allowances[_from][msg.sender], "Not enough funds allowed");
-        _allowances[_from][msg.sender] = _allowances[_from][msg.sender].sub(_value);
+        require(_value <= allowances[_from][msg.sender], "Not enough funds allowed");
+        allowances[_from][msg.sender] = allowances[_from][msg.sender].sub(_value);
```

Comment:
No changes should be made to `OldLegacyRepToken` or `LegacyRepToken` as they are meant to represent contracts live on mainnet.  Even though this particular change may just be internal details, I think it is generally a bad practice to be mutating those two contracts.